### DISCUSSION TITLE
LDAP Allow invalid SSL Certificate option

### DISCUSSION THREAD

**Discussion by JonsBackLog** [Leantime/leantime#2216] [Q&A]
I'm using leantime in a local internal network configuration and although my domain controller has an ssl certificate its a self generated and not official one. Other applications usually have a "Allow invalid SSL Certificate" option in their configuration files. Does leantime have something similar that I can use as I don't see anything in the documentation. 

Essentially I keep getting the error message "'Strong(er) authentication required" on ldap. I've tried altering the code in the LDAP module itself to ignore the cert check but to no avail. Still the same error. tried adding ldaps:// as others have suggested. I have no issues with Snipe-it and bookstack setup with ldap configuration. Not sure why leantime is giving so many issues.

**Comment by MIPMHannes** [comment]
I am using the docker container. There is file you need to configure to disable cert check. See [here](https://github.com/Leantime/leantime/issues/2200#issuecomment-1898202025). Maybe Dev can fix this if we ask nicely?
